docs/extensions/tutorials/_posts/1970-01-13-UsingLocalization.md

```diff
@@ -0,0 +1,220 @@
+
+Think back to [Getting Started]({{ site.}}/docs/extensions/tutorials/getting-started). We made a simple extension with a line of text in one screen:
+
+```JavaScript{5}
+#file: tom.restaurants/app/screens/List.js
+export default class List extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text style={styles.text}>Let's eat!</Text>
+      </View>
+    );
+  }
+}
+```
+
+To implement translations to that we'd have to import `I18n`:
+
+```JavaScript{11}
+#file: tom.restaurants/app/screens/List.js
+import React, {
+  Component
+} from 'react';
+
+import {
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+
+import { I18n } from 'shoutem.i18n';
+```
+
+Then use the imported `I18n`:
+
+```JavaScript{2}
+#file: tom.restaurants/app/screens/List.js
+<View style={styles.container}>
+  <Text style={styles.text}>I18n.t(ext('letsEatMessage'))</Text>
+</View>
+```
+
+Now we need to implement the fallback by creating a `translations` directory in the `app` segment:
+
+```ShellSession
+$ mkdir app/translations && touch app/translations/en.json
+```
+
+And adding our extension's string to it:
+
+```JavaScript
+#file: tom.restaurants/app/translations/en.json
+{
+  "tom": {
+    "restaurants": {
+      "letsEatMessage": "Let's eat!"
+    }
+  }
+}
+```
+
+This string has to be exported in `app/index.js`:
+
+```JavaScript{1,10-16}
+#file: tom.restaurants/app/index.js
+import enTranslations from './translations/en.json';
+
+// Constants `screens` (from extension.js) and `reducer` (from index.js)
+// are exported via named export
+// It is important to use those exact names
+
+// export everything from extension.js
+export * from './extension';
+
+export const tom = {
+  i18n: {
+    translations: {
+      en: enTranslations,
+    },
+  },
+};
+```
+
+And that's it!
+
+#### Pluralized Strings
+
+When editing strings that contain pluralization (e.g. `1 point` vs. `2 points`) we utilize the following format for the language file:
+
+```JSON{4-7}
+#file: en.json
+{
+  "shoutem": {
+    "loyalty": {
+      "pointsInStore": {
+        "one": "{% raw %}{{{% endraw %}count}} point collected.",
+        "other": "{% raw %}{{{% endraw %}count}} points collected.",
+        "zero": "No points collected."
+      }
+    }
+  }
+}
+```
+
+And the following method inside the actual React Native component:
+
+```JavaScript{6}
+#file: shoutem.loyalty/app/components/PlaceIconView.js
+const { place, points, onPress } = this.props;
+
+return (
+  <Caption>
+    {I18n.t(ext('pointsInStore'), { count: points })}
+  </Caption>
+);
+```
+
+It's important to use the `count` variable name specifically due to the way `i18n-js` functions.
+
+#### Variables in Strings
+
+When editing strings that contain variables we utilize the following format for the language file:
+
+```JSON{3-4}
+#file: en.json
+{
+  "shoutem": {
+    "auth": {
+      "loggedInUserInfo": "Username: {% raw %}{{{% endraw %}username}}"
+    }
+  }
+}
+```
+
```

Comment:
Explain `{% raw %}{{{% endraw %}`. Refer to docs of i18n-js how this works.

Reply:
That has nothing to do with i18n-js. It's not visible in the code snippet on shoutem.github.io. It's used to show double brackets in the code snippets, because if there is not `{% raw %}` and `{% endraw %}`, the site tries to load a variable instead, which doesn't exists so you end up with a blank space.